rep-0003.rst

```diff
@@ -233,6 +233,66 @@ Buildsystem support:
 
   - build from source
 
+Lunar Loggerhead (May 2017 - May 2019)
+--------------------------------------
+Required Support for:
+
+- Ubuntu Xenial (16.04)
+- Ubuntu Yakkety (16.10)
+- Ubuntu Zesty (17.04)
+
+Recommended Support for:
+
+- Debian Stretch
+- Fedora 25
+
+Architecture Supported:
+
+- amd64
+- arm32
+- arm64
+
+Minimum Requirements:
+
+- C++11
+
+  - GCC 4.9 on Linux, as it's the version that Debian Jessie ships with
+
+- Python 2.7
+
+  - Python 3.4 not required, but testing against it is recommended
```

Comment:
This is a pointless recommendation. Very few maintainers know how to do it. 

For a serious recommendation, we would need a link to detailed instructions explaining exactly how to run Lunar with Python 3.4.